### ISSUE TITLE
Add term Thickened cortex of the femur

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10451]
**Preferred term label:**
Thickened cortex of the femur

**Synonyms**
Broad cortex of femur- Cortical thickening of the femur- Thickened cortices of femur

**Definition (free text, please give PubMed ID)**
Abnormal thickening of the cortex of the femur
PMID24389367

**Parent term (use hpo.jax.org/app)**
Thickened cortex of long bones HP:0000935
Abnormal femur morphology HP:0002823

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Caffey Disease (omim114000), for example: PMID24390061

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000810
- label: Thickened femoral cortex
- parents: [Thickened fibular cortex (HP:6000705)](https://hpo.jax.org/app/browse/term/HP:6000705) | [Abnormal femur morphology (HP:0002823)](https://hpo.jax.org/app/browse/term/HP:0002823)
- pmid: PMID:24390061
- def: Abnormally increased thicknes of the cortical layer of the femur.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10451
- synonyms:
